docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -143,6 +143,16 @@ BigCommerce sends requests for tax estimates on product item, shipping, and hand
 
 [View the Estimate Taxes API reference](/docs/rest-contracts/tax#estimate-taxes).
 
+### Performance
+
+Merchants expect fast tax estimate responses to facilitate a streamlined checkout experience for their shoppers, hence performance is an essential consideration for a tax provider.
+
+To protect the shopper checkout experience from overly slow tax responses, tax estimate requests will timeout after a set period of time. The default threshold of four seconds is intended to support a wide variety of use cases (e.g. large tax estimate requests).
```

Comment:
> The default threshold of four seconds

While this is factual, and following on with some of @aleachjr's thinking, we have an opportunity here to reduce the 'default' timeout on new providers to a lower amount.

I could imagine a case for making this _two seconds_, for example, and removing the note about "You may request a lower timeout". This would put us in a stronger position going forward, and not inviting additional support chatter.

That is, because we're documenting this publicly we have a good opportunity to reconsider our position.

Another related consideration: if we want to explore how we can better service small carts vs large carts we might end up with a solution that somehow dynamically changes the timeout based on the number of items in the cart/quote request.

Reply:
I'm going to run the numbers here on `AvalaraProvider` to see how they are actually tracking with regard to the general range of request timings.

Reply:
According to our recent metrics, our p99 for Avalara Provider is maxing-out around the 3 seconds mark, along with a couple of other providers (TaxCloud, Thomson Reuters). Meanwhile, the p99 mean is 1.15s for Avalara, though a substantial 2.26 for TaxCloud.

Vertex is also up there at 2.7s maximum (664ms mean) on p99.

p95 for our tracked platform providers, meanwhile, is mostly under 1 second (TaxCloud is currently an outlier at over 2, though a longer timeframe shows it mellow out also).

It's worth noting that these are to be regarded as higher grade tax providers, as opposed to the many limited availability providers we have on the platform - providers which generally have a far lower grade of support/technical implementation.

So, with that data up our sleeve and the context around the calibre of tax providers I think we can proceed with this documentation update as a starting point and address improvements as a separate initiative - in particular looking into what's going on with the p99 on our core providers (e.g. are these large carts?)